boards/realtek/rts5912_evb/doc/rts5912_evb.rst

```diff
@@ -0,0 +1,91 @@
+.. _rts5912_evb:
+
+Realtek RTS5912 Evaluation Board
+################################
+
+Overview
+********
+The RTS5912 EVB is a development platform to evaluate the Realtek RTS5912 embedded controller.
+
+.. figure:: rts5912evb.webp
+   :width: 800px
+   :align: center
+   :alt: RTS5912-GR EVB
+
+Hardware
+********
+- Realtek-M300 Processor (compatible to Cortex-M33)
+- Memory:
+
+   - 384KB SRAM
+   - 64KB ROM
+   - 512KB Flash(MCM)
+   - 256B Battery SRAM
+- PECI interface 3.1
+- FAN, PWM and TACHO pins
+- 6x I2C instances
+- eSPI header
+- 1x PS/2 ports
+- Keyboard interface headers
+
+For more information about the evb board please see `RTS5912_EVB_Schematics`_ and `RTS5912_DATASHEET`_
+
+The board is powered through the +5V USB Type-C connector or adaptor.
+
+
+Supported Features
+==================
+The RTS5912 EVB supports the following hardware features:
+
++-----------+------------+-------------------------------------+
```

Comment:
@kartben is this how it should be or should it be using https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nordic/nrf5340dk/doc/index.rst?plain=1#L26 ?